scripts/index-files.js

```diff
@@ -2,6 +2,7 @@
 
 const { Files } = require('../utils/files')
 
+
```

Comment:
Why the extra empty line here?

Reply:
there's supposed to be 2 blank lines after the import/require